worldedit-core/src/main/java/com/fastasyncworldedit/core/extent/processor/MultiBatchProcessor.java

```diff
@@ -26,9 +31,14 @@
 
 public class MultiBatchProcessor implements IBatchProcessor {
 
-    private IBatchProcessor[] processors;
+    private static final Logger LOGGER = LogManagerCompat.getLogger();
+
     private final LoadingCache<Class<?>, Map<Long, Filter>> classToThreadIdToFilter =
             FaweCache.IMP.createCache((Supplier<Map<Long, Filter>>) ConcurrentHashMap::new);
+    private boolean[] faweExceptionReasonsUsed = new boolean[FaweException.Type.values().length];
```

Comment:
This should be an (Enum)Set

Reply:
I used an array because it's not going to throw errors with concurrent read/write, and it's not an important enough "feature" to need to worry about perfect synchronisation - the only thing that would happen is an error being displayed twice, but as long as it's not being spammed, it's fine.